README.md

```diff
@@ -82,1045 +82,1045 @@ ostorlab scan run --install -g agent_group.yaml domain-name example.com
+| CVE-2023-36033   | ❌         | Not remotely exploitable/User interaction needed. |
+| CVE-2023-36036   | ❌         | Not remotely exploitable/User interaction needed. |
+| CVE-2023-36563   | ❌         | Not remotely exploitable/User interaction needed. |
+| CVE-2023-36584   | ❌         | Not remotely exploitable/User interaction needed. |
+| CVE-2023-36761   | ❌         | Not remotely exploitable/User interaction needed. |
+| CVE-2023-36802   | ❌         | Not remotely exploitable/User interaction needed. |
+| CVE-2023-36844   | ✅         | Added to Asteroid.                                |
+| CVE-2023-36845   | ✅         | Added to Asteroid.                                |
+| CVE-2023-36846   | ✅         | Added to Asteroid.                                |
+| CVE-2023-36847   | ✅         | Added to Asteroid.                                |
+| CVE-2023-36851   | ✅         | Added to Asteroid.                                |
+| CVE-2023-36874   | ❌         | Not remotely exploitable/User interaction needed. |
+| CVE-2023-36884   | ❌         | Not remotely exploitable/User interaction needed. |
+| CVE-2023-37450   | ❌         | Not remotely exploitable/User interaction needed. |
+| CVE-2023-37580   | ✅         | Nuclei Template was used.                         |
+| CVE-2023-38035   | ✅         | Nuclei Template was used.                         |
+| CVE-2023-38180   | ❌         | Not remotely exploitable/User interaction needed. |
+| CVE-2023-38205   | ✅         | Nuclei Template was used.                         |
+| CVE-2023-38606   | ❌         | Not remotely exploitable/User interaction needed. |
+| CVE-2023-38831   | ❌         | Not remotely exploitable/User interaction needed. |
+| CVE-2023-40044   | ✅         | Metasploit module was used.                       |
+| CVE-2023-41061   | ❌         | Not remotely exploitable/User interaction needed. |
+| CVE-2023-41064   | ❌         | Not remotely exploitable/User interaction needed. |
+| CVE-2023-41179   | ❌         | Not remotely exploitable/User interaction needed. |
+| CVE-2023-41763   | ❌         | Not remotely exploitable/User interaction needed. |
+| CVE-2023-41991   | ❌         | Not remotely exploitable/User interaction needed. |
+| CVE-2023-41992   | ❌         | Not remotely exploitable/User interaction needed. |
+| CVE-2023-41993   | ❌         | Not remotely exploitable/User interaction needed. |
+| CVE-2023-4211    | ❌         | Not remotely exploitable/User interaction needed. |
+| CVE-2023-42793   | ✅         | Nuclei Template was used.                         |
+| CVE-2023-42824   | ❌         | Not remotely exploitable/User interaction needed. |
+| CVE-2023-44487   | ❌         | Not remotely exploitable/User interaction needed. |
+| CVE-2023-46604   | ⏳         |                                                   |
+| CVE-2023-46747   | ⏳         |                                                   |
```

Comment:
```suggestion
| CVE-2023-46747   | ✅           | Nuclei Template was used.                         |
```
can you change this please so my PR can be merged directly with no conflict

Reply:
Done